.github/ISSUE_TEMPLATE/documentation-issue.md

```diff
@@ -0,0 +1,11 @@
+**Is your feature request related to a problem?**
+A new feature has been added.
+
+**What solution would you like?**
```

Comment:
These words looks good for feature/bug fix, should we tune the words for documentation?